pkg/daemon/discover/discover.go

```diff
@@ -167,7 +167,9 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 	// return any add or remove events, but none that match device mapper
 	// events. string matching is case-insensitve
 	events := make(chan string)
-	go rawUdevBlockMonitor(events, []string{"(?i)add", "(?i)remove"}, []string{"(?i)dm-[0-9]+"})
+	go rawUdevBlockMonitor(events,
+		[]string{"(?i)add", "(?i)remove"},
+		[]string{"(?i)dm-[0-9]+", "(?i)rbd[0-9]+"})
```

Comment:
Could this be made configurable by an environment variable?

There are probably some users having other block devices that should not be picked up (like rbd).

Reply:
@galexrt yes, and we really should. though, there are some questions related to what form those options should take, and resolving that is going to take a bit more time.

Reply:
@noahdesu @galexrt Hey. I've been looking at this issue and would want to work on this. So can you elaborate on this environment variable and how can udevfilter be configured with an environment variable.